docs/pages/admin-guides/infrastructure-as-code/terraform-starter/enroll-resources.mdx

```diff
@@ -370,6 +370,7 @@ Replace the following placeholders:
 |-----------------------|------------------------------------------------------------------------------|
 | AWS_REGION            | The AWS region where you will deploy Agents, e.g., `us-east-2`               |
 | PROXY_SERVICE_ADDRESS | The host and port of the Teleport Proxy Service, e.g., `example.teleport.sh:443` |
+| "terraform-identity/identity" | The path to the identity file created for Terraform. [Terraform Provider Auth](../../../reference/terraform-provider.mdx)  |
```

Comment:
```suggestion
```

I think this row was left over from an earlier change. Since we include the `tctl terraform env` command in Step 3, there's no need to update the identity file path.

In the current change, I don't think there's enough context to understand the purpose of the "Terraform Provider Auth" link.

Reply:
ok, I'll remove the identity file references. That should also solve this auth conflict. Thanks!